package.json

```diff
@@ -65,8 +67,10 @@
     "jest": "^24.8.0",
     "lint-staged": "^9.4.3",
     "mockdate": "^2.0.5",
+    "moment": "^2.24.0",
     "prettier": "^1.19.1",
     "pretty-quick": "^1.11.1",
+    "react-dates": "^21.8.0",
```

Comment:
react-dates relies on moment, which unfortunately means that right now we have both moment and dayjs. We can optimize later.

Reply:
Funny, moment is generally the standard we had been using within our Javascript within the UI Framework (prior to OWAs or spa).  Then when doing OWA developent, date-fns became the preferred format, so we switched to what.  And then SPA chose dayjs.  And now we have to do back to moment to use React-Dates.

I had been perfectly happy with moment, *but* I believe a drawback of it is that it is all-or-nothing... you need to bundle the whole library in your app, not just the components you need?

Reply:
Yeah, date-fns and dayjs seem like the good options. [This](https://github.com/you-dont-need/You-Dont-Need-Momentjs#brief-comparison) is a helpful comparison.